WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -244,6 +244,20 @@ class PostCoordinator: NSObject {
             try? post.managedObjectContext?.save()
         }
     }
+
+    /// Cancel active and pending automatic uploads of the post.
+    func cancelAutoUploadOf(_ post: AbstractPost) {
+        cancelAnyPendingSaveOf(post: post)
+
+        // nil-ing this out will make the `shouldAttemptAutoUpload` property to return false.
```

Comment:
This was updated in Jans PR: https://github.com/wordpress-mobile/WordPress-iOS/blob/0e42cff404559bcc9bcc0b759f6ed08ea339fec0/WordPress/Classes/Services/PostCoordinator.swift#L277

Reply:
Yeah, I figured there are going to be some conflicts. 😄

Reply:
I was wrong! I picked the wrong branch. I should have been targeting #12352. Thank you for the heads up. Marking this as Not Ready for Review for now to avoid the confusion.